cpp/doxygen/developer_guide/TESTING.md

```diff
@@ -52,7 +55,7 @@ Testing header-only APIs requires CUDA compilation so should be done in `.cu` fi
 
 ## Base Fixture
 
-All libcuspatial unit tests should make use of a GTest 
+All libcuspatial unit tests should make use of a GTest
```

Comment:
I think here we should encourage the use of `cuspatial::test::BaseFixture`?
https://github.com/rapidsai/cuspatial/blob/branch-23.06/cpp/include/cuspatial_test/base_fixture.hpp

Reply:
I think that should be a separate PR.